webapp/components/DonationInfo/DonationInfo.spec.js

```diff
@@ -37,7 +37,7 @@ describe('DonationInfo.vue', () => {
     ).toBe('donations.donate-now')
   })
 
-  it('creates a title from the current month and a translation string', () => {
+  it.skip('creates a title from the current month and a translation string', () => {
```

Comment:
Hey @mattwr18 can you tell what's the matter with the tests?

Reply:
@roschaefer I added it in my commit message. These tests are testing functionality that is no longer used in production. It was failing the build, and instead of refactoring, I've left the tests, but skipped them till we decide whether to remove completely or reuse the donations functionality